File: fabric/src/main/java/com/ultreon/mods/advanceddebug/fabric/AdvancedDebugFabric.java
package com.ultreon.mods.advanceddebug.fabric;

import com.ultreon.mods.advanceddebug.AdvancedDebug;
import com.ultreon.mods.advanceddebug.client.Config;
import net.fabricmc.api.ClientModInitializer;

public class AdvancedDebugFabric implements ClientModInitializer {
    @Override
    public void onInitializeClient() {
        AdvancedDebug advancedDebug = new AdvancedDebug();
        advancedDebug.init();

        Config.register(null);
    }
}

File: fabric/src/main/java/com/ultreon/mods/advanceddebug/fabric/AdvancedDebugFabric.kt
package com.ultreon.mods.advanceddebug.fabric

import com.ultreon.mods.advanceddebug.AdvancedDebug
import com.ultreon.mods.advanceddebug.client.Config
import dev.architectury.utils.EnvExecutor
import net.fabricmc.api.EnvType
import net.fabricmc.api.ModInitializer

object AdvancedDebugFabric : ModInitializer {
    override fun onInitialize() {
        EnvExecutor.runInEnv(EnvType.CLIENT) {
            Runnable {
                AdvancedDebug.init()
                Config.register(null)
            }
        }
    }
}